packages/transports/google-cloud-logging/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@loglayer/transport-google-cloud-logging",
   "description": "Google Cloud Logging (Stackdriver) transport for the LogLayer logging library.",
-  "version": "2.1.2",
+  "version": "3.0.0",
   "type": "module",
```

Comment:
changeset will do the version bump before publish

may i ask you to roll this change back?